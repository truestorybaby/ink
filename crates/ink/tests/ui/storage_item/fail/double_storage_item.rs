#[ink::storage_item(derive = false)]
#[ink::storage_item(derive = true)]
#[derive(Default)]
struct Contract<KEY: StorageKey = ManualKey<123>> {
    a: u16,
    b: u64,
    c: u128,
}

fn main() {}
