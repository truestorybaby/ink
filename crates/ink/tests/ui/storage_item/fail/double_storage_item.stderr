error: only one `ink::storage_item` is allowed
 --> tests/ui/storage_item/fail/double_storage_item.rs:2:1
  |
2 | #[ink::storage_item(derive = true)]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
